docs/configuration.md

```diff
@@ -1094,43 +1094,117 @@ var config = {
 >
 > ```js
 > {
->   adBreaks: Array<PKAdBreakObject>,
->   playAdsAfterTime?: number
+>   playAdsAfterTime?: number,
+>   adBreaks: Array<PKAdBreakObject>
 > }
 > ```
 >
 > ##### Default: `-`
 >
 > ##### Description: Defines an advertising scheme (optional).
 >
+> ##
+>
+> > ### config.advertising.playAdsAfterTime
+> >
+> > ##### Type: `number`
+> >
+> > ##### Default: `config.playback.startTime`
+> >
+> > ##### Description: Only play ad breaks scheduled after this time (in seconds). This setting is strictly after - e.g. setting playAdsAfterTime to 15 will cause the player to ignore an ad break scheduled to play at 15s.
+> >
 > > ##
 > >
 > > ### config.advertising.adBreaks
 > >
 > > ##### Type: `Array<PKAdBreakObject>`
 > >
+> > ##### Description: The ad breaks scheme
+> >
+> > ##
+> >
 > > > ##### Type `PKAdBreakObject`
 > > >
 > > > ```js
 > > > {
 > > >   position: number,
+> > >   percentage?: number, // optional
+> > >   every?: number, // optional
 > > >   ads: Array<PKAdObject>
 > > > }
 > > > ```
 > > >
-> > > > ##### Type `PKAdObject`
+> > > ##
+> > >
+> > > > ##### `PKAdBreakObject.position`
 > > > >
-> > > > ```js
-> > > > {
-> > > >   url?: Array<string>,
-> > > >   response?: Array<string>,
-> > > >   bumper?: boolean
-> > > > }
-> > > > ```
-> >
-> > ##### Default: `[]`
+> > > > ##### Type: `number`
+> > > >
+> > > > ##### Description: The ad break position in seconds.
+> > > >
+> > > > ##
+> > > >
+> > > > ##### `PKAdBreakObject.percentage`
+> > > >
+> > > > ##### Type: `number`
+> > > >
+> > > > ##### Description: The ad break position in percentage of the media length instead of by seconds (optional).
+> > > >
+> > > > ##
+> > > >
+> > > > ##### `PKAdBreakObject.every`
+> > > >
+> > > > ##### Type: `number`
+> > > >
+> > > > ##### Description: Play ad break every X seconds (optional).
+> > > >
+> > > > ##
+> > > >
+> > > > ##### `PKAdBreakObject.ads`
+> > > >
+> > > > ##### Type: `Array<PKAdObject>`
+> > > >
+> > > > ##### Description: An array of ads to play (Ad pod).
+> > > >
+> > > > ##
+> > > >
+> > > > > ##### Type `PKAdObject`
+> > > > >
+> > > > > ```js
+> > > > > {
+> > > > >   url?: Array<string>,
+> > > > >   response?: Array<string>,
+> > > > >   bumper?: boolean
+> > > > > }
+> > > > > ```
+> > > > >
+> > > > > ##
+> > > > >
+> > > > > > ##### `PKAdObject.url`
+> > > > > >
+> > > > > > ##### Type: `Array<string>`
+> > > > > >
+> > > > > > ##### Description: List of urls, each one specifies the ad tag url that is requested from the ad server. The player will request the first url, if failed, it request the second and so on (aka waterfalling).
```

Comment:
```suggestion
> > > > > > ##### Description: List of urls, each one specifies the ad tag url that is requested from the ad server. The player will request the first url, if failed, it will request the second url and so on (aka waterfalling).
```